resources/lib/utilities.py

```diff
@@ -181,40 +182,49 @@ def findEpisodeMatchInList(id, seasonNumber, episodeNumber, list, idType):
     return {}
 
 
-def convertDateTimeToUTC(toConvert):
-    if toConvert:
-        dateFormat = "%Y-%m-%d %H:%M:%S"
-        try:
-            naive = datetime.strptime(toConvert, dateFormat)
-        except TypeError:
-            naive = datetime(*(time.strptime(toConvert, dateFormat)[0:6]))
-
-        try:
-            local = naive.replace(tzinfo=tzlocal())
-            utc = local.astimezone(tzutc())
-        except ValueError:
-            logger.debug(
-                'convertDateTimeToUTC() ValueError: movie/show was collected/watched outside of the unix timespan. Fallback to datetime utcnow')
-            utc = datetime.utcnow()
-        return str(utc)
-    else:
-        return toConvert
-
-
-def convertUtcToDateTime(toConvert):
-    if toConvert:
-        dateFormat = "%Y-%m-%d %H:%M:%S"
-        try:
-            naive = dateutil.parser.parse(toConvert)
-            utc = naive.replace(tzinfo=tzutc())
-            local = utc.astimezone(tzlocal())
-        except ValueError:
-            logger.debug(
-                'convertUtcToDateTime() ValueError: movie/show was collected/watched outside of the unix timespan. Fallback to datetime now')
-            local = datetime.now()
-        return local.strftime(dateFormat)
-    else:
-        return toConvert
+def to_datetime(value):
```

Comment:
I replaced the existing datetime transformation methods with these 4 new ones, the iso8601 methods come from the trakt.py library.